lib/ui/myplot/PlotListItem.dart

```diff
@@ -15,9 +15,19 @@ class PlotListItemViewModel {
   final ImageURLProvider imageProvider;
   final ViewModelProvider<PlotDetailViewModel> detailViewModelProvider;
 
-
-  PlotListItemViewModel({ String title, String subtitle, String detail, double progress, ImageURLProvider imageProvider, ViewModelProvider<PlotDetailViewModel> provider}) : this.title = title, this.subtitle = subtitle, this.detail = detail, this.progress = progress, this.imageProvider = imageProvider, this.detailViewModelProvider = provider;
-
+  PlotListItemViewModel(
```

Comment:
I know that this file is not part of this PR, I've just changed the NetworkImageFromFuture widget and made a format code. @LeeMatthewHiggins If you think that you will have conflicts, let me know and I'll revert this file and change only the NetworkImageFromFuture widget

Reply:
👍